ops/helmfiles/dagster/migrate.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# Migrates our dagster cluster DB schema
+# (taken from https://docs.dagster.io/deployment/guides/kubernetes/how-to-migrate-your-instance)
+
+# Get the names of Dagster's Dagit and Daemon Deployments created by Helm
+export DAGIT_DEPLOYMENT_NAME=`kubectl -n dagster get deploy \
+    --selector=component=dagit -o jsonpath="{.items[0].metadata.name}"`
+export DAEMON_DEPLOYMENT_NAME=`kubectl -n dagster get deploy \
+    --selector=component=dagster-daemon -o jsonpath="{.items[0].metadata.name}"`
+
+# Save each deployment's replica count to scale back up after migrating
+export DAGIT_DEPLOYMENT_REPLICA_COUNT=`kubectl -n dagster get deploy \
+    --selector=component=dagit -o jsonpath="{.items[0].status.replicas}"`
+export DAEMON_DEPLOYMENT_REPLICA_COUNT=`kubectl -n dagster get deploy \
+    --selector=component=dagster-daemon -o jsonpath="{.items[0].status.replicas}"`
+
+echo "scaling down..."
```

Comment:
design preference: i like to output status messages in shell scripts to stderr, to make it easier to parse script output if we decide to let other scripts consume this one in the future.